frontend/src/static/js/v_authorship.js

```diff
@@ -99,19 +102,24 @@ window.vAuthorship = {
       encodeHash();
     },
 
-    expandAll(isActive) {
-      const renameValue = isActive ? 'file active' : 'file';
-
-      const files = document.getElementsByClassName('file');
-      Array.from(files).forEach((file) => {
-        file.className = renameValue;
+    expandAll() {
+      this.selectedFiles.forEach((file) => {
+        file.active = true;
+        file.wasCodeLoaded = true;
       });
+    },
 
-      this.activeFilesCount = isActive ? this.selectedFiles.length : 0;
+    collapseAll() {
+      this.selectedFiles.forEach((file) => {
+        file.active = false;
+      });
     },
 
-    updateCount() {
-      this.activeFilesCount = document.getElementsByClassName('file active').length;
+    toggleCollapsed(file) {
```

Comment:
perhaps `toggleFile` would be a better method name? as the file may not be collapsed

Reply:
Would `toggleFileActive` be fine?

Reply:
how about `toggleFileActiveProperty`?

Reply:
Sounds good :+1: